nixos/doc/manual/contributing-to-this-manual.chapter.md

```diff
@@ -2,6 +2,8 @@
 
 The [DocBook] and CommonMark sources of the NixOS manual are in the [nixos/doc/manual](https://github.com/NixOS/nixpkgs/tree/master/nixos/doc/manual) subdirectory of the [Nixpkgs](https://github.com/NixOS/nixpkgs) repository.
 
+See the [nixpkgs documentation syntax](https://nixos.org/manual/nixpkgs/unstable/#sec-contributing-markup).
```

Comment:
```suggestion
The sources of the NixOS manual are in the [nixos/doc/manual](https://github.com/NixOS/nixpkgs/tree/master/nixos/doc/manual) subdirectory of the [Nixpkgs](https://github.com/NixOS/nixpkgs) repository. It uses the same CommonMark variant as [Nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#sec-contributing-markup).
```

Reply:
We intentionally do not make any corrections other than the modifications that are necessary for linking. Corrections that have to do with docbook will be in another PR.

Reply:
Well, I think just saying “see the nixpkgs documentation syntax” does not really connect – why should one see it?

Reply:
How about now, @jtojnar .